crates/holochain_p2p/src/types/event.rs

```diff
@@ -100,6 +100,21 @@ impl From<&actor::GetActivityOptions> for GetActivityOptions {
     }
 }
 
+/// Multiple ways to fetch op data
+#[derive(Debug, derive_more::From)]
+pub enum FetchOpDataQuery {
+    /// Fetch all ops with the hashes specified
+    Hashes(Vec<holo_hash::DhtOpHash>),
+    /// Fetch all ops within the time and space bounds specified
+    Region {
+        /// The min and max timestamp to fetch
+        time_bounds: (Timestamp, Timestamp),
+        /// The min and max DhtLocation to fetch
+        // TODO: make an ArcInterval or something
+        location_bounds: (DhtLocation, DhtLocation),
```

Comment:
Should we try to use the [RangeBounds](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=2681f5812814dad78f710859c5a12a72) trait here?

Reply:
There are actually some more appropriate types available in my `quantized-gossip` branch which this will be merged into, so I'll leave it like this for now since it is just a placeholder